octobot_trading/api/exchange.py

```diff
@@ -46,6 +45,10 @@ def get_exchange_names() -> list:
     return Exchanges.instance().get_exchange_names()
 
 
+def get_exchange_channel(chan_name, exchange_name) -> ExchangeChannel:
```

Comment:
How this API method is used ?

Reply:
To subscribe to order channels in : https://github.com/Drakkar-Software/OctoBot-Notifications/pull/2
```python
    async def _subscribe_to_order_channels(self):
        order_channel_name = OrdersChannel.get_name()
        try:
            for exchange_name in get_exchange_names():
                channel = get_exchange_channel(order_channel_name, exchange_name)
                await channel.new_consumer(self._order_notification_callback)
        except KeyError:
            self.logger.error("No order channel to subscribe to: impossible to send order notifications")
```

Reply:
We could also add an API to subscribe to the order channel with a callback, that would wrap the internal wiring of octobot-trading but would imply a new API for each other future subscription

Reply:
I really don't like to interact with exchange channel outside of OctoBot-Trading.

Can we imagine to pass the callback as the parameter of the API method like :
```
def subscribe_to_order_channels(callback):
    for exchange_name in get_exchange_names():
        channel = get_exchange_channel(order_channel_name, exchange_name)
        await channel.new_consumer(self._order_notification_callback) 
```

Reply:
Yep, I'll add that